--- wildmidi-0.2.3.4/src/Makefile.am.orig	2010-07-30 02:33:46.000000000 +0200
+++ wildmidi-0.2.3.4/src/Makefile.am	2010-12-30 17:28:32.846114393 +0100
@@ -15,7 +15,7 @@
 lib_LTLIBRARIES=	libWildMidi.la
 libWildMidi_la_SOURCES=		wm_error.c file_io.c reverb.c lock.c gus_pat.c wildmidi_lib.c
 libWildMidi_la_CFLAGS=		$(PREFER_NON_PIC)
-libWildMidi_la_LDFLAGS=		-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -shared $(LDFLAGS) -no-undefined -export-symbols-regex '^WildMidi.*'
+libWildMidi_la_LDFLAGS=		-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) $(LDFLAGS) -no-undefined -export-symbols-regex '^WildMidi.*'
 libWildMidi_la_LIBADD=      -lm -lc
 
 bin_PROGRAMS= wildmidi
